libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -712,7 +711,7 @@ bool PbftManager::stateOperations_() {
 bool PbftManager::updateSoftVotedBlockForThisRound_() {
   if (soft_voted_block_for_this_round_) {
     // Have enough soft votes for current round already
-    return false;
+    return true;
```

Comment:
How can you just change false to true without changing the way we check updateSoftVotedBlockForThisRound_() return value in code ? Was this bug or what ?

Reply:
Yes, this is a bug.
PBFT certify stage is a pulling stage, within 2 lambda time that will loop in every 100ms until node cert vote on a block.
Return false will cause only be able to go in certify stage once here https://github.com/Taraxa-project/taraxa-node/blob/reward_cert_votes/libraries/core_libs/consensus/src/pbft/pbft_manager.cpp#L889 That's not right